app/rest/jsondb/src/main/java/io/syndesis/jsondb/impl/JsonRecord.java

```diff
@@ -22,13 +22,13 @@
 
     private final String path;
     private final String value;
-    private final int kind;
+    private final String ovalue;
```

Comment:
what do we gain by replacing int with String? seems to me like that would have negative performance

Reply:
Type was removed.  odata is a new field used for something different